src/main/java/DbConnection/JdbcConnector.java

```diff
@@ -0,0 +1,25 @@
+package DbConnection;
+
+import com.mchange.v2.c3p0.ComboPooledDataSource;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+
+public class JdbcConnector {
+    final Properties properties = new Properties();
+
+    private void getcon() throws IOException {
```

Comment:
We haven't any constraints about variable length. We can name this method "getConnection()" for example.